sponsorships/2025-07.csv

```diff
@@ -3,7 +3,7 @@ Abse2001,200,"Weekly scores: [2 (06/25-07/01), 2 (07/02-07/08)]"
 Anshgrover23,100,"Weekly scores: [2 (06/25-07/01), 1 (07/02-07/08)]"
 ArnavK-09,500,"Weekly scores: [3 (06/25-07/01), 3 (07/02-07/08)]"
 MustafaMulla29,75,"Weekly scores: [1 (06/25-07/01), 1 (07/02-07/08)]"
-ShiboSoftwareDev,300,"Weekly scores: [2 (06/25-07/01), 3 (07/02-07/08)]"
+ShiboSoftwareDev,500,"Weekly scores: [3 (06/25-07/01), 3 (07/02-07/08)]"
```

Comment:
@ShiboSoftwareDev doesn't this seem wrong? Isn't it supposed to be 500 + X? You had two three star weeks